<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CailWindow</class>
 <widget class="QMainWindow" name="CailWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>233</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="pushButton_self">
           <property name="text">
            <string>self</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_cail">
           <property name="text">
            <string>cail</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_Acc">
           <property name="text">
            <string>Acc</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QPushButton" name="pushButton_level">
           <property name="text">
            <string>level</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_up">
           <property name="text">
            <string>up</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_down">
           <property name="text">
            <string>down</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_left">
           <property name="text">
            <string>left</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_right">
           <property name="text">
            <string>right</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_back">
           <property name="text">
            <string>back</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_in_m1">
      <property name="text">
       <string>IN_M1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_in_m2">
      <property name="text">
       <string>IN_M2</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>586</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
